frontend/src/container/ListOfDashboard/DashboardsList.tsx

```diff
@@ -438,6 +438,7 @@ function DashboardsList(): JSX.Element {
 				const getLink = (): string => `${ROUTES.ALL_DASHBOARD}/${dashboard.id}`;
 
 				const onClickHandler = (event: React.MouseEvent<HTMLElement>): void => {
+					event.stopPropagation();
```

Comment:
why it is required here ?

Reply:
this click handler is also used directly in the action popover - for the `view` option

Reply:
got it thanks!